core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -383,61 +379,68 @@ async def show_address(
     *,
     address_qr: str | None = None,
     case_sensitive: bool = True,
-    title: str = "Confirm address",
+    path: str | None = None,
+    account: str | None = None,
     network: str | None = None,
     multisig_index: int | None = None,
     xpubs: Sequence[str] = (),
-    address_extra: str | None = None,
     title_qr: str | None = None,
```

Comment:
`title_qr` is now unused, both here and in `TR`, so we could delete it completely

Reply:
And at the same time, we can delete `address_extra` from `TR::show_address`

Reply:
fixed in [4b5ae11](https://github.com/trezor/trezor-firmware/pull/2843/commits/4b5ae117c480ccb7954996cabb233d367d2d14d6)